test/thrift/parser/parser_test.exs

```diff
@@ -918,17 +918,14 @@ defmodule Thrift.Parser.ParserTest do
 
   describe "namespace option" do
     setup do
-      path = Path.join(@test_file_dir, "namespace.thrift")
+      path = Path.join(@test_file_dir, "module.thrift")
```

Comment:
I renamed this to make the test code below a little clearer. There were too many things named "namespace" before.